test/stablecoin_converter.js

```diff
@@ -32,16 +40,17 @@ contract("StablecoinConverter", async (accounts) => {
   })
 
   // Basic Trade used in most of the tests:
-  // Trade for user_1: amount of token_1 sold: 20000, amount of token_2 bought: feeSubtracted(10000),
-  // Trade for user_2: amount of token_2 sold: feeSubtracted(10000), amount of token_1 bought: feeSubtracted(feeSubtracted(10000) * 2)
+  // Trade for user_1: amount of token_1 sold: 20020, amount of token_2 bought: 10000,
```

Comment:
Shouldn't the old example have still worked, and just required different volumes to be passed in the solution?

Reply:
Because of different rounding, actually, it did not pass. So I kept the same example, it is roughly the same volumes and it is the same price